lib/travis/api/v3/result.rb

```diff
@@ -2,7 +2,7 @@ module Travis::API::V3
   class Result
     attr_accessor :access_control, :type, :resource, :status, :href, :meta_data, :warnings
 
-    def initialize(access_control, type, resource = [], status: 200, **meta_data)
+    def initialize(access_control, type, resource, status: 200, **meta_data)
```

Comment:
The upgrade from ruby 2.1 to 2.2 changed how params work. So in this version of ruby, passing something like this [:lint, object] for type and resource confused ruby between the optional resource param and the **meta_data hash. Reviewed this with @rkh and decided to remove the << method that modified the resource after instantiation as well.